pontoon/sync/vcs/repositories.py

```diff
@@ -173,6 +173,9 @@ def commit(self, path=None, message=None, user=None, branch=None):
         if code != 0:
             raise CommitToRepositoryException(text_type(error))
 
+        if type(error) is bytes:
+            error = error.decode('utf-8')
```

Comment:
Why do you only add the check for the `commit()` method of the `CommitToGit` class, and not also other `CommitToX` classes?

Reply:
Ah, that's because this is the only one I tested for, and I didn't think to look at the other classes. Thanks!